### ISSUE TITLE
`Chrome 50` browserslist config removes rest params but doesn't include polyfill

### ISSUE THREAD

**Issue by mischnic** [swc-project/swc#2718]
### Describe the bug

With a `Chrome 50` browserslist, the `...rest` parameter is just removed from the function definition even though it's used.

Also Chrome 47 appears to support them (https://caniuse.com/rest-parameters), but swc transpiles them away even with `chrome 48`?

### Input code

```typescript
export const foo = (...stuff) => {
  console.log("STUFF", stuff);
};
```

### Config

```
{
  "devDependencies": {
    "@swc/cli": "0.1.51",
    "@swc/core": "1.2.107",
  },
  "browserslist": "chrome 50",
}
```

```json
{
	"env": {},
	"sourceMaps": true
}
```

### Expected behavior

Currently, this happens:
```js
export const foo = ()=>{
    console.log("STUFF", stuff);
};
```

### Version

1.2.107

### Additional context

https://github.com/parcel-bundler/parcel/issues/7284

**Comment by kdy1** [comment]
I think ideal fix for this is accepting json file.
Will see if it's simple task.

**Comment by Austaras** [comment]
For the second part of the problem, see [here](https://github.com/kangax/compat-table/blob/gh-pages/data-es6.js). So chrome 47 do support rest parameter but not until chrome 49 is rest destructing supported

**Comment by Austaras** [comment]
closed by #2975

**Comment by kdy1** [comment]
Thanks!
